test/web.test.js

```diff
@@ -16,32 +16,30 @@ test('Web resources belong to a single web', function(t) {
 
   var w1 = nap.web()
     , w2 = nap.web()
-    , fn = function() {}
+    , fn = function() { t.ok(true, 'resource added to w1') }
```

Comment:
Are these (and following) test changes related to the changes above, or just an improvement of the tests?